File: fibonacci.c
#include <stdio.h>
int main()
/*el lenguaje C corresponde al paradigma de la programacón Estructurada o modular  */
{
int n, a=0,b=1,temp,i;
printf("ingrese hasta que Nro hacer Fibonacci: ");
scanf("%d",&n);
printf("%d %d ",a,b);
for(i=2;i<n;i++){
    temp= a+b;
    a=b;
    b= temp;
    printf("%d", temp);
}
return 0;
}

File: Fibonacci.cs
using System;
 
namespace fibonacci
{
	class Program
	{
		static void Main(string[] args)
		{
			byte CAN, K;
			int A, B, C;
			string linea;
 
			Console.Write("hasta que Nro desea hacer fibonacci: ");
			linea = Console.ReadLine();
 
			CAN = byte.Parse(linea);
			A = 1;B = 1;
			Console.Write(A + " " + B + " ");
			for(K = 3; K <= CAN; K++){
				C = A + B;
				Console.Write(C + " ");
 
				//intercambio de valores
				A = B;
				B = C;
			}
			Console.WriteLine();
		}
	}
}
